openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1065,70 +1096,91 @@ Authorization: BEARER czZCaGRSa3F0MzpnWDFmQmF0M2JW
              "UniversityDegreeCredential"
            ]
          },
-         "proof":{
-            "proof_type":"jwt",
-            "jwt":"eyJ0eXAiOiJvcGVuaWQ0dmNpL...Lb9zioZoipdP-jvh1WlA"
-         }
+         "proofs": [
+            {
+               "proof_type":"jwt",
+               "jwt":"eyJ0eXAiOiJvcGVuaWQ0dmNpL...Lb9zioZoipdP-jvh1WlA"
+            },
+            {
+               "proof_type":"jwt",
+               "jwt":"eyJraWQiOiJkaWQ6ZXhhbXBsZ...KPxgihac0aW9EkL1nOzM"
+            }
+         ]
```

Comment:
this is the most simple solution(mirrored from credential endpoint suggestion).
Imo for ldp, you could still use JSON objects as proof array values?
```suggestion
         "proof_type":"jwt",
         "proofs": [
            "eyJ0eXAiOiJvcGVuaWQ0dmNpL...Lb9zioZoipdP-jvh1WlA",
            "eyJraWQiOiJkaWQ6ZXhhbXBsZ...KPxgihac0aW9EkL1nOzM"
         ]
```